spec/models/ability_spec.rb

```diff
@@ -37,7 +38,18 @@
     expect(ability.can?(:create, build(:discussion))).to be_falsy
   end
 
+  it 'logged in user can post comments' do
+    comment = build(:comment, author: user)
+    expect(can?(:create, comment)).to be_truthy
+  end
+
+  it 'anonymous is not able to post comments' do
+    ability = Ability.new(nil)
+    expect(ability.can?(:create, build(:comment))).to be_falsy
+  end
+
   def can?(action, subject)
     ability.can?(action, subject)
   end
+
```

Comment:
Similar as above. See this [rubycop check](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/mixin/empty_lines_around_body.rb)